src/docs/content/builders/toast.md

```diff
@@ -18,8 +18,110 @@ description: A succinct message that is displayed temporarily.
 
 ## Usage
 
-To create a toast, use the `createToast` builder function. You can then reference the anatomy or
-example above to create your toast.
+Unlike most builders, the toast is not component-based. Instead, it provides a global functionality
+that can be accessed from anywhere in your application. To accomplish this, it is recommended that
+you call the builder function `createToaster` inside the lib directory. This will make the function
+accessible to all components in your application.
+
+The first step is to create a `Toast` component that will be used to render toast notifications. We
```

Comment:
I'd call it Toaster tbh

Reply:
Yes now that you say it, Toaster seems better!